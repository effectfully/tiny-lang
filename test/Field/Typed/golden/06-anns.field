let ?bool    = T : bool;
let field'   = 1 : field;
let #vector  = { T } : vector;