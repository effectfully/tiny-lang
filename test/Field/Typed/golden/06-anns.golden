Program (Statements [ELet (UniVar {_uniVarUni = Bool, _uniVarVar = ?bool_3}) (EConst (UniConst Bool True)),ELet (UniVar {_uniVarUni = Field, _uniVarVar = field'_4}) (EConst 1),ELet (UniVar {_uniVarUni = Vector, _uniVarVar = #vector_5}) (EConst (UniConst Vector [True]))])